accord-core/src/test/java/accord/messages/PreAcceptTest.java

```diff
@@ -196,7 +196,8 @@ void multiKeyTimestampUpdate() throws ExecutionException
             messageSink.assertHistorySizes(0, 1);
             Assertions.assertEquals(ID3, messageSink.responses.get(0).to);
             PartialDeps expectedDeps = new PartialDeps(Ranges.of(range(0, 12)), KeyDeps.NONE, RangeDeps.NONE);
-            Assertions.assertEquals(new PreAccept.PreAcceptOk(txnId2, Timestamp.fromValues(1, 110, ID1), expectedDeps),
+            Timestamp expectedTs = Timestamp.fromValues(1, 110, ID1).withExtraFlags(txnId2.flags());
```

Comment:
what change caused this?  I get the following locally (only changed this test, nothing else)

```
Expected :PreAcceptOk{txnId:[1,50,2,3], witnessedAt:[1,110,2,1], deps:[Range(0, 12]]:{}, {}}
Actual   :PreAcceptOk{txnId:[1,50,2,3], witnessedAt:[1,110,0,1], deps:[Range(0, 12]]:{}, {}}
```